_posts/2020-08-07-g2p.md

```diff
@@ -148,11 +152,63 @@ mappings:
     mapping: eng_ipa_to_arpabet.json 
 ```
 
-### Special settings
+If you're not familiar with yaml, and you're not just copy pasting from here, I recommend having a look at one of the many [tutorials](https://gettaurus.org/docs/YAMLTutorial/) on how to use yaml properly before attempting to write your own mapping configuration.
+
+====
+***PROPOSED END OF FIRST POST***
+====
+
+## Advanced: A deeper dive into writing tricky rules
+
+You may have noticed that the rules described above for converting words like 'dog' and 'cat' to IPA are woefully incomplete. The real world use cases for `g2p` often need to account for a lot more messiness than was described in the artificial example above. In fact, for languages like English, `g2p` is likely **not** a good solution. The English writing system is notoriously inconsistent, and there already exist a variety of other tools that account for all of the lexical (word-specific) idiosyncracies in deriving the IPA-form from the orthographic form. For many Indigenous languages, the writing system is sufficiently close to the spoken form that `g2p` is a very appropriate solution. In the following sections, I'll describe some common problems when writing rules, and how to fix them.
+
+### Rule Ordering
+
+The order of your rules in `g2p` really matters! This is because some rules can either create or remove the context for other rules to apply. In linguistics, these rule ordering patterns are usually talked about as either [feeding, bleeding, counter-feeding, or counter-bleeding](https://linguistics.stackexchange.com/questions/6084/whats-the-difference-between-counterbleeding-bleeding-and-feeding) relationships. There are potentially valid reasons to want to encode any of these types of relationships in your rules. 
```

Comment:
```suggestion
**The order of your rules in `g2p` really matters**! This is because some rules can either create or remove the context for other rules to apply. In linguistics, these rule ordering patterns are usually talked about as either [feeding, bleeding, counter-feeding, or counter-bleeding](https://linguistics.stackexchange.com/questions/6084/whats-the-difference-between-counterbleeding-bleeding-and-feeding) relationships. There are potentially valid reasons to want to encode any of these types of relationships in your rules. 
```

I'm suggesting bolding here, because I didn't understand this, and it hurt me :(

I would even put this in the "basic" tutorial, rather than the "deep-dive" tutorial.